src/index.html

```diff
@@ -1,14 +1,32 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="moyo.jpg" alt="Logo">
+      </a>
+      <nav>
+        <ul class="box">
+          <li class="box-item"><a class="box-link" href="#">apple</a></li>
+          <li class="box-item"><a class="box-link" href="#">samsung</a></li>
+          <li class="box-item"><a class="box-link" href="#">smartphones</a></li>
+          <li class="box-item is-active" data-qa="hover"><a class="box-link" href="#">laptops & computers</a></li>
```

Comment:
the link should have `is-active` class `hover` data-qa attribute:

![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/8b46f9a1-1199-4997-a437-13c59b291ef3)